src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
```

Comment:
@tchaffee @tbushman just saw this comment, which contradicts the one I left below. Whoops! I didn't see this before.

Normally, I think following DRY is good, but in this case, we're only repeating the string once. I don't think that's a big deal and it removes some clutter with variables. I think the original way is fine, unless I'm missing another use of `dropdown` where we'd be repeating the string again. The way it is currently is also less desirable because we're confusing things with 2 similar variable names.